### PR TITLE
Resprites the CentCom armour suit, and adds mask + helmet to a suit storage thingy

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

The old CentCom armour outfit looked like shite, and was old as hell. It's not been updated to have the Cap's armour suit values and an edited version of the Cap's suit sprite, with a complimentary green.

Also adds a suit storage unit that contains a unique gas mask as well, because why not.

fsafsafa

## Why It's Good For The Game

Old armour = cringe, looked bad
New armour = very sexy, looks good

and a suit storage thingy is good for anyone who wants to add it to a map per se, idk.

## Changelog
:cl:
add: Added new CC armour suit, with accompanying helmet and gasmask
add: Added suit storage that contains this new armour and gasmask.
expansion: Expands on the old CC armour suit
imageadd: added some sprites for da drip
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/armor.dm
Comment (Tennessee116) [inline code/modules/clothing/suits/armor.dm:235]: A comically generic description.

Comment (Tennessee116) [reply code/modules/clothing/suits/armor.dm:235]:
```suggestion
	desc = "A tactical suit designed for Nanotrasen's most prestigious."
```

Comment (axietheaxolotl) [reply code/modules/clothing/suits/armor.dm:235]: That was the description of the old Centcom armour, I just didn't change it 'cause it was 7 am and I hadn't slept lol.

**Review by Twaticus — CHANGES_REQUESTED** [review]
please remove oldcentcom from mob/clothing/suit & obj/clothing/suits 
(oldcaparmor could be removed aswell if youd be so kind to 🥺 )

**Comment by axietheaxolotl** [comment]
> 
> 
> please remove oldcentcom from mob/clothing/suit & obj/clothing/suits
> (oldcaparmor could be removed aswell if youd be so kind to 🥺 )

donezo

**Comment by Time-Green** [comment]
We're in a feature freeze for the month of august, so chances are this'll get closed

I think resprites alone are fine, check here for info #60608

**Review by Twaticus — APPROVED** [review]

**Comment by axietheaxolotl** [comment]
mfw merge conflict

**Comment by axietheaxolotl** [comment]
ok freeze is over
maintainers pls

**Comment by LemonInTheDark** [comment]
> 
> 
> ok freeze is over
> maintainers pls

I let you live

**Comment by LemonInTheDark** [comment]
@Twaticus do you want this? waiting on code review?

**Comment by axietheaxolotl** [comment]
maintainers PLEASE
I beg of you

**Comment by LemonInTheDark** [comment]
Well no matter what you've gotta wait for the prefs menu to be done, since we're on a freeze around that

File: code/modules/clothing/masks/gasmask.dm
Comment (LemonInTheDark) [inline code/modules/clothing/masks/gasmask.dm:109]: Should give it a unique description instead of copying the captain's directly

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

**Comment by LemonInTheDark** [comment]
prefs menu is done, so well

**Comment by LemonInTheDark** [comment]
Let's jive

**Pull request merged**
